pkg/cache/cache.go

```diff
@@ -12,6 +14,10 @@ import (
 	"k8s.io/client-go/tools/cache"
 )
 
+const (
+	resyncDefault = 5
```

Comment:
Default interval is 5ns?

Upstream's default resync interval also has a random skew applied; that would probably be a good idea to incorporate here.

Reply:
Do you know where that skew code is Brad ? We had considered doing the same. 
I read this as 5 minutes, but that is also bad, why is it not still 10 hours ? 5 minutes would make everything bunches up and worse.

Reply:
For Duration the base integer unit is nanosecond I believe; usually stuff gets parsed from a string or you multiply something by `time.Hour` etc.

I'll see if I can dig up a link to the upstream code.

Reply:
Mb, I had this set to 5 for testing will change. This is in minutes though; I'm multiply with minutes below.